### DISCUSSION TITLE
Spindle speed control

### DISCUSSION THREAD

**Discussion by Speghettification** [winder/Universal-G-Code-Sender#2643] [General]
UGS doesn't change spindle speed using M4.  I am trying to run a JTech laser, and the beam intensity is tied to the S command.  Program sends S1M4, but the spindle speed on DRO is either maxed out (13000rpm) of off.  I can change it with the override down to 10% (1300).  Can anyone offer advice?  THX

**Comment by breiler** [comment]
What firmware and version are you using and what are the firmware settings (retrieved with `$$` on GRBL based machines)?
